ceph.spec.in

```diff
@@ -464,6 +464,8 @@ Summary:        Utility to bootstrap Ceph clusters
 BuildArch:      noarch
 Requires:       lvm2
 Requires:       python%{python3_pkgversion}
+Requires:       openssh-server
+Requires:       which
```

Comment:
Please avoid adding a dependency on which. It's not standard or strictly needed.
https://lwn.net/Articles/874049/

Reply:
@phlogistonjohn this was needed because otherwise `cephadm bootstrap` failed in the CentOS 8 container with this `which: command not found` error. I initially thought this was caused by `shutil.which` but reviewed the [source code](https://github.com/python/cpython/blob/3.6/Lib/shutil.py) and the implementation doesn't use the `which` command, so not sure who's actually calling it there.